pybv/io.py

```diff
@@ -36,6 +36,7 @@
 
 
 def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
+                      overwrite=False,
                       events=None, resolution=0.1, unit='µV',
                       fmt='binary_float32', meas_date=None):
```

Comment:
What about using Black-ish formatting here, which will make it easier to see the parameters?

```
def write_brainvision(
    *, data, sfreq, ch_names, fname_base, folder_out,
    overwrite=False,
    events=None,
    resolution=0.1,
    unit='µV',
    fmt='binary_float32',
    meas_date=None
):